action.yml

```diff
@@ -22,6 +22,10 @@ inputs:
     description: Folder in the workspace in which the .git repo resides
     required: false
     default: ""
+  bumpoverride:
+    description: Override the version bump, can be either 'major', 'minor' or 'patch'
+    required: false
+    default: ""
```

Comment:
the default value should be `auto`, I changed from empty-string in `version-bumper` before deploying, you can see here https://github.com/TomerFi/version-bumper/commit/e67b5057850e23ba28655ae9b17976df64580b06.